bin/faucet/src/client.rs

```diff
@@ -113,6 +114,7 @@ impl FaucetClient {
             .executor
             .execute_transaction(self.id, 0, &[], transaction_args)
             .map_err(|err| {
+                error!("{err}");
                 FaucetError::InternalServerError(format!("Failed to execute transaction: {}", err))
```

Comment:
Was logging not working correctly here? Unless there is a good reason, I'd rather not add this ad-hoc. Maybe worth creating an issue for this?

Reply:
Created https://github.com/0xPolygonMiden/miden-node/issues/499